src/jobsautoreport/slack.py

```diff
@@ -0,0 +1,123 @@
+import logging
+from typing import Any
+
+from slack_sdk import WebClient
+
+logger = logging.getLogger(__name__)
+
+
+class SlackReporter:
+    def __init__(self, web_client: WebClient, channel_id: str) -> None:
+        self._client = web_client
+        self._channel_id = channel_id
+
+    def send_message(self, data: dict[str, Any]) -> None:
+        if not self._is_valid(data):
+            return
+
+        blocks_list = self.format_as_message(data)
+        res = self._client.chat_postMessage(
+            channel=self._channel_id, blocks=blocks_list
+        )
+        ts = res["ts"]
```

Comment:
have you found the signification for `ts`?

Reply:
Adding now